### PR TITLE
[Standalone] use only 1 public ip address

### PR DESCRIPTION
**Pull request by JosepSampe**
Refactor to use only 1 public ip address in standalone mode
 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

**Comment by gilv** [comment]
@JosepSampe i was thinking that many concurrent jobs need to be supported by Lirhops at the same time. Then they all need reuse the same initial VM with the same floating IP. Initial VM doesn't run compute by itself, but only start point to deploy the job snd create other VMs per job. There might be 100 of users, each submit a job. In this case, Lithops will use the same floating IP of the same VM to deploy job to 100 VMs created by Lithops for this scpeficied job. I just wonder how we know when this special VM finished all jobs and can be dismantled.

File: config/README.md [hunk 1]
```diff
@@ -204,7 +204,4 @@ fexec = lithops.FunctionExecutor(rabbitmq_monitor=True)
 |standalone | soft_dismantle_timeout | 300 |no| Time in seconds to stop the VM instance after a job **completed** its execution |
 |standalone | hard_dismantle_timeout | 3600 | no | Time in seconds to stop the VM instance after a job **started** its execution |
 |standalone | exec_mode | consume | no | If set to  **create** standalone backend will automatically create VMs based on the standalone backend|
-|standalone | disable_log_monitoring | False | no | If set to  True pull remote logs will be disabled. This can improve running time|
-|standalone | local_runtime_load | False | no | If set to  True the assumption is that compute backend VM contains required docker image locally. This saves docker pull or load|
-|standalone | use_http | false | no | `true` or `false`. Whether or not use http connections to communicate with the VM instance. If `falase` (default) it communicates with the VM using ssh connections. If true, it uses the encryption_key to encrypt the payload sent trough http.|
```
Comment (gilv) [inline config/README.md:209]: why "use_http", "disable_log_monitoring" were removed?

Comment (JosepSampe) [reply config/README.md:209]: yes, not needed anymore

Comment (gilv) [reply config/README.md:209]: @JosepSampe how it will know if to load runtime from the image or use docker pull?

Comment (gilv) [reply config/README.md:209]: @JosepSampe ahh...i see it now "pull_runtime"

Comment (JosepSampe) [reply config/README.md:209]: yep, I changed the variable name to `pull_runtime` since this variable only activate/deactivate the command `docker pul ...`

**Comment by gilv** [comment]
@JosepSampe what is the status of it?

**Comment by JosepSampe** [comment]
@gilv Working hard on it ;) I may finish tomorrow.

**Comment by gilv** [comment]
Need some testing strategy, otherwise might miss thing. Will add more tests

- [ ] New runtime, vanilla image (non custom), job size of 1 VM

**Comment by gilv** [comment]
@JosepSampe one quick comment..i think we need to add a special profile name for master node. Currently config has a single entry `profile_name: bx2-8x32` that is used by master and slaves..but master basically does nothing, so it can be smallest config available, like bx2-2x8'. Are you fine to add `profile_master_name: bx2-2x8` ?

**Comment by JosepSampe** [comment]
> @JosepSampe one quick comment..i think we need to add a special profile name for master node. Currently config has a single entry `profile_name: bx2-8x32` that is used by master and slaves..but master basically does nothing, so it can be smallest config available, like bx2-2x8'. Are you fine to add `profile_master_name: bx2-2x8` ?

@gilv I thought exactly the same. Do you want to keep `profile_name` and add `master_profile_name`? or do you prefer `worker_profile_name` and `master_profile_name`?

**Pull request merged**